MetaMask/metamask-extension
# Update segment instantiation check. Only check if SEGMENT_WRITE_KEY exists

## Explanation

We can remove the `SEGMENT_HOST` check for our developer environments. If `SEGMENT_HOST` does not exist, then the `analytics-node` library will resolve the host to a default host. 

## Manual testing steps

1. Setup related environment variables
  a. Go to the `.metamaskrc` file in our repo
  b. Ensure `SEGMENT_WRITE_KEY` is set to developer, Segment write key
  c. Ensure `SEGMENT_HOST` is not set
2. Restart developer environment with `yarn start` 
3. Ensure you have no ad blockers running (uBlock Origin, AdBlock)
4. Open the MetaMask Extension
5. Ensure you are Participating in MetaMetrics in "Settings > Security & Privacy"
6. Open Segment and verify that events are being tracked
